flink-ml-benchmark/src/main/resources/robustscaler-benchmark.json

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "version": 1,
+  "robustscaler10000000": {
+    "inputData": {
+      "className": "org.apache.flink.ml.benchmark.datagenerator.common.DenseVectorGenerator",
+      "paramMap": {
+        "vectorDim": 100,
+        "colNames": [
+          [
+            "features"
+          ]
+        ],
+        "seed": 2,
+        "numValues": 10000000
+      }
+    },
+    "stage": {
+      "className": "org.apache.flink.ml.feature.robustscaler.RobustScaler",
+      "paramMap": {
+        "inputCol": "features",
```

Comment:
Would it be simpler to remove this line and use the default value? Same for `outputCol`.

Same for similar parameters used in other benchmark json files added in this PR.

Reply:
I prefer to keep the `inputCol` because it shows the relationship between `DataGenerator` and `Stage`. As for `outputCol` we can remove it.

Reply:
How about we change the column name from `features` to `inputCol` for simplicity and consistency with other tests?

Reply:
Actually, there's no consistency within existing tests. Currently, we have both `input` and `features` as the column name. To keep consistency, I'll change all of them to `input`, is that acceptable to you?